src/test/java/com/jayway/asyncservlet/ApplicationTests.java

```diff
@@ -30,6 +39,22 @@ public void shouldPass_springBootRepo() throws Exception {
         log.info(result.toString());
 
         assertTrue(op.getStatusCode().is2xxSuccessful());
-
+    }
+    
+    @Import(Application.class)
```

Comment:
wasn't sure whether you still wanted to skip the api call, so just piggy-backed on the test. you're right though, if it stays, it should be in a separate class.